src/common/dataHooks/useCharacters.js

```diff
@@ -56,11 +58,9 @@ export function useCharacterPartialUpdate() {
 
   const partialUpdateCharacter = async (formData) => {
     const properties = {
-      relatedAudio: formData?.relatedAudio || [],
-      relatedImages: formData?.relatedImages || [],
-      relatedVideos: formData?.relatedVideos || [],
+      ...relatedMediaForApi({ formData }),
       note: formData?.generalNote || '',
-      relatedDictionaryEntries:
+      related_dictionary_entries:
```

Comment:
Is this snake_case required on the backend? We should fix it for you

Reply:
Yes, it is. That would be great.